sb-cb-ui-utils-v2/library/sunbird-cb/utils-v2/src/lib/pipes/pipe-public-URL/pipe-public-URL.pipe.ts

```diff
@@ -9,9 +9,12 @@ export class PipePublicURL implements PipeTransform {
     this.environment = environment
   }
   transform(value: string): any {
-    const mainUrl = value && value.split('/content').pop() || ''
-    const finalURL = `${this.environment.contentHost}/${this.environment.contentBucket}/content${mainUrl}`
-    return value ? finalURL : ''
+    if (value.includes("karmayogi")) {
```

Comment:
This should check for content urls which has karmayogi. Not all URLs which has karmayogi

Reply:
@christyfernandes  As we discussed, I have updated the comments.